edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Core/EdgeHubConnectionException.cs

```diff
@@ -1,17 +1,22 @@
-﻿// Copyright (c) Microsoft. All rights reserved.
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
 namespace Microsoft.Azure.Devices.Edge.Hub.Core
 {
     using System;
+    using System.Collections.Generic;
```

Comment:
Why were these usings added?

Reply:
I remember there is a compile error in 1 file that System.Collections.Generic using statement can't resolve properly and that's why I added a rule to add this by default.

Reply:
But this file doesn't need that anything from SystemCollections.Generic, as far as I can tell... also why is formatting causing compilation failures?